src/solvers/flattening/bv_pointers.h

```diff
@@ -104,6 +104,34 @@ class bv_pointerst:public boolbvt
   postponed_listt postponed_list;
 
   void do_postponed(const postponedt &postponed);
+
```

Comment:
If you are making changes to this PR some documentation here might be nice.

Reply:
I arguably should have done this right away. Now fixed.